api/src/task/notifications/opportunity_notifcation.py

```diff
@@ -276,11 +278,28 @@ def _get_last_notified_versions(
 
     def _build_description_fields_content(self, description_change: dict, opp_id: int) -> str:
         after = description_change["after"]
-        if after:
-            description_section = SECTION_STYLING.format("Description")
-            description_section += f"{BULLET_POINTS_STYLING} The description has changed.<br>"
-            return description_section
-        return ""
+        if not after:
+            return ""
+
+        description_section_parts = [
+            SECTION_STYLING.format("Description"),
+            f"{BULLET_POINTS_STYLING} <i>New Description:</i>",
+        ]
+
+        if len(after) > TRUNCATION_THRESHOLD:
+            truncated = after[:TRUNCATION_THRESHOLD]
+
+            if contains_regex(truncated, r"<[^>]+>"):
+                truncated = truncate_html_safe(truncated)
+
```

Comment:
Might be a naming thing, but should `truncate_html_safe` take in the truncation length?

I'm imagining I'd want to use it like so:
```py
my_html = "..."

truncated_html = truncate_html_safe(my_html, TRUNCATION_THRESHOLD)
```
And move the bit you have here to that utility function.